modules/core/src/main/java/org/apache/ignite/cache/CacheMetrics.java

```diff
@@ -657,7 +657,7 @@
     public boolean isStatisticsEnabled();
 
     /**
-     * Checks whether management is enabled on this cache.
+     * Checks whether cache management via MBean is enabled.
```

Comment:
To be honest, both versions look good to me. The original javadoc comes from JSR 107 - 12.2. MXBean Definitions.
`/**
* Checks whether management is enabled on this cache.
* <p>
* The default value is <code>false</code>.
*
* @return true if management is enabled
*/
boolean isManagementEnabled();`

So, it is up to you.